src/Microsoft.ML.Transforms/Text/TextTransform.cs

```diff
@@ -34,8 +34,8 @@ namespace Microsoft.ML.Runtime.Data
     // A transform that turns a collection of text documents into numerical feature vectors. The feature vectors are counts
     // of (word or character) ngrams in a given text. It offers ngram hashing (finding the ngram token string name to feature
     // integer index mapping through hashing) as an option.
-    /// <include file='doc.xml' path='doc/members/member[@name="TextTransform"]/*' />
-    public sealed class TextTransform : IEstimator<ITransformer>
+    /// <include file='doc.xml' path='doc/members/member[@name="FeaturizeTextEstimator"]/*' />
+    public sealed class FeaturizeTextEstimator : IEstimator<ITransformer>
```

Comment:
>FeaturizeTextEstimator [](start = 24, length = 22)

I guess this should be `TextFeaturizationEstimator`

Reply:
Would that be `TextFeaturizationEstimator`, or `TextFeaturizingEstimator`? Do we care about being consistent? I see elsewhere we're naming things `ColumnConcatenatingEstimator`.

Reply:
`TextFeaturizingEstimator` is more consistent, I agree. Also, it probably won't hurt to rename the file

---
In reply to: [227052881](https://github.com/dotnet/machinelearning/pull/1328#discussion_r227052881) [](ancestors = 227052881)

Reply:
Let me do the file renames on another PR. to avoid further churn here 

---
In reply to: [227053961](https://github.com/dotnet/machinelearning/pull/1328#discussion_r227053961) [](ancestors = 227053961,227052881)